content/pt-br/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -18,122 +15,91 @@ princípios do Kubernetes, notavelmente o [ciclo de controle](/docs/concepts/#ku
-     para fornecer armazenamento de base de dados durável, um *StatefulSet* para executar *SampleDB* e
-     um *Job* para lidar com a configuração inicial.
-   * Se você apagá-lo, o Operador tira um *snapshot* e então garante que
-     o *StatefulSet* e *Volumes* também são removidos.
-6. O operador também gere backups regulares da base de dados. Para cada recurso *SampleDB*,
-  o operador determina quando deve criar um *Pod* que possa se conectar
-   à base de dados e faça backups. Esses *Pods* dependeriam de um *ConfigMap*
-   e / ou um *Secret* que possui detalhes e credenciais de conexão com à base de dados.
-7. Como o Operador tem como objetivo fornecer automação robusta para o recurso
-   que gere, haveria código de suporte adicional. Para este exemplo,
-   O código verifica se a base de dados está a executar uma versão antiga e, se estiver,
-   cria objetos *Job* que o atualizam para si.
-
-## Instalar Operadores
-
-A forma mais comum de instalar um Operador é a de adicionar a
-definição personalizada de recurso (*Custom Resource Definition*) e
-o seu Controlador associado ao seu cluster.
-O Controlador vai normalmente executar fora do
-{{< glossary_tooltip text="plano de controle" term_id="control-plane" >}},
-como você faria com qualquer aplicação containerizada.
+4. Código do controlador que consulta o plano de controle para descobrir quais recursos *SampleDB* estão configurados.
+5. O núcleo do Operador é o código que informa ao servidor da API (*API server*) como fazer com que a realidade corresponda aos recursos configurados.
+   * Se você adicionar um novo *SampleDB*, o operador configura *PersistentVolumeClaims* para fornecer armazenamento durável da base de dados, um *StatefulSet* para executar o *SampleDB* e um *Job* para lidar com a configuração inicial.
+   * Se você excluir um *SampleDB*, o operador faz um *snapshot* e em seguida, garante que o *StatefulSet* e os *Volumes* também sejam removidos.
+6. O operador também gerencia backups regulares da base de dados. Para cada recurso *SampleDB*, o operador determina quando criar um *Pod* que pode se conectar ao banco de dados e fazer backups. Esses *Pods* dependeriam de um *ConfigMap* e/ou um *Secret* que tenha detalhes da conexão e credenciais do banco de dados.
+7. Como o operador tem como objetivo fornecer automação robusta para o recurso que gerencia, haveria código de suporte adicional. Para este exemplo, o código verifica se o banco de dados está a executando uma versão antiga e, se estiver, cria objetos *Job* que fazem a atualização para você.
+
+## Implantando operadores
+
+A maneira mais comum de implantar um operador é adicionar a definição personalizada de recurso (*Custom Resource Definition*) e o Controlador associado ao seu cluster.
+O Controlador normalmente é executado fora do {{< glossary_tooltip text="plano de controle" term_id="control-plane" >}}, assim como você executaria qualquer aplicação containerizada.
```

Comment:
"camada de gerenciamento" instead of "plano de controle". 

"Container" is an english word. What do you think about "conteinerizada"?